### PR TITLE
Add Documentation about how to avoid org conflict

### PR DESCRIPTION
**Pull request by AdrieanKhisbe**
built on #3277 cf #3562.

Based on personal experience, how I struggled with org&spacemacs, and
finally got it to work :)

### CODE REVIEW

File: doc/FAQ.org [hunk 1]
```diff
@@ -92,6 +92,18 @@ repository instead of the one shipped with emacs. Then, any `org` related code
 should not be loaded before `dotspacemacs/user-config`, otherwise both versions
 will be loaded and will conflict.
 
+Because of autoloading, calling to `org` function will trigger the loading up
+of the `org` shipped with emacs wich will induce conflicts.
+One way to do do your `org` config is to wrap you config code in a
```
Comment (nixmaniack) [inline doc/FAQ.org]: delete one of the "do"s :)

File: layers/org/README.org [hunk 2]
```diff
@@ -38,13 +38,23 @@ This layer enables  [[http://orgmode.org/][org mode]] for Spacemacs.
...
-repository instead of the one shipped with emacs. Then, any `org` related code
-should not be loaded before `dotspacemacs/user-config`, otherwise both versions
+Since version 0.104, spacemacs uses the =org= version from the org ELPA
+repository instead of the one shipped with emacs. Then, any =org= related code
+should not be loaded before =dotspacemacs/user-config=, otherwise both versions
 will be loaded and will conflict.
 Org layer checks if this has occurred and signals an error if org features have
 been instantiated prior to org layer loading.
 
+One way to do do your =org= config is to wrap you config code in a
```
Comment (nixmaniack) [inline layers/org/README.org]: and here as well!

File: doc/FAQ.org (hunk 1 above)
Comment (AdrieanKhisbe) [inline doc/FAQ.org]: typo fixed :)

**Comment by AdrieanKhisbe** [comment]
@nixmaniack Typo fixed :)

**Comment by syl20bnr** [comment]
Thank you !
Cherry-picked into develop branch, you can safely delete your branch.

**Comment by AdrieanKhisbe** [comment]
@syl20bnr de rien :)
(*you're welcome*)

**Pull request closed without merging**